<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>800</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Image Processing</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="selectImageButton1">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>20</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Select Image 1</string>
                </property>
            </widget>
            <widget class="QPushButton" name="selectImageButton2">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>80</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Select Image 2</string>
                </property>
            </widget>
            <widget class="QPushButton" name="selectImageButton3">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>140</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Select Image 3</string>
                </property>
            </widget>
            <widget class="QPushButton" name="selectImageButton4">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>200</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Select Image 4</string>
                </property>
            </widget>
            <widget class="QPushButton" name="processButton">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>260</y>
                        <width>150</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Process</string>
                </property>
            </widget>
            <widget class="QLabel" name="imageLabel">
                <property name="geometry">
                    <rect>
                        <x>200</x>
                        <y>20</y>
                        <width>300</width>
                        <height>300</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="scaledContents">
                    <bool>true</bool>
                </property>
            </widget>
            <widget class="QLabel" name="binaryImageLabel">
                <property name="geometry">
                    <rect>
                        <x>520</x>
                        <y>20</y>
                        <width>300</width>
                        <height>300</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Binary Image</string>
                </property>
                <property name="scaledContents">
                    <bool>true</bool>
                </property>
            </widget>
            <widget class="QLabel" name="decompressedImageLabel">
                <property name="geometry">
                    <rect>
                        <x>740</x>
                        <y>20</y>
                        <width>300</width>
                        <height>300</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Decompressed Image</string>
                </property>
                <property name="scaledContents">
                    <bool>true</bool>
                </property>
            </widget>
            <widget class="QTextEdit" name="outputTextEdit">
                <property name="geometry">
                    <rect>
                        <x>200</x>
                        <y>340</y>
                        <width>840</width>
                        <height>400</height>
                    </rect>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
